src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj

```diff
@@ -66,6 +66,7 @@
       TODO: add OptimizeAssemblies="lib/netstandard2.0/System.Threading.Tasks.Extensions.dll"
     -->
     <ExpectedDependency Include="System.Threading.Tasks.Extensions" />
+    <ExpectedDependency Include="Microsoft.Bcl.AsyncInterfaces"/>
```

Comment:
@tmat since this package is being inserted outside of Roslyn I guess we actually need to put this not in this list but rather in the `_Dependency` list farther down?